src/app/home/userOffers/UserOffers.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import List from "@/components/list/List";
+import {
+  useGetBorrowedBooksQuery,
+  useGetBorrowedFromMeQuery,
+  useGetUserMarketBooksQuery,
+} from "@/api/marketApi";
+import { MarketBook } from "@/interfaces/MarketBook";
+import ListItem from "@/components/listItem/ListItem";
+
+export default function UserOffers() {
+  const { data: borrowedBooks } = useGetBorrowedBooksQuery();
+  const { data: borrowedFromMeBooks } = useGetBorrowedFromMeQuery();
+  const { data: myBooks } = useGetUserMarketBooksQuery({});
+
+  return (
+    <>
+      {myBooks ? (
```

Comment:
It'd be good to have a few empty states here:

- "Encouragement" states whenever some section is empty, e.g. `No books here yet` + button to start borrowing/add a book
- A global state when there's nothing here yet. May include a link to other page, buttons as above or some kind of sheperd/tutorial UI}

BTW - something to consider, if you want to play with a new library:  (https://github.com/shipshapecode/shepherd) ;)